### PR TITLE
Added work and about me sections to index.html 

### PR DESCRIPTION
**Pull request by c00p75**
Added work and about me sections to index.html which used grid and flexbox | aligned about me section with flexbox | used <ul> for categories in about myself and Work sections | Added necessary image files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,52 +1,314 @@
...
+							</p>
+						</div>
+					</div> 
+				</div>
+
+				<div class="work_item">
+					<img src="images/Snapshoot Portfolio (1).png" alt="snapshot portifolio image">
+					<div class="work_details">
+						<div class="project">
+							<p class="work_item_header">Multi-Post Stories</p>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: Please make sure you use `hx` tag for All the project card headers. A `p` tag is not semantically appropriate. Please change it to a heading

File: index.html [hunk 2]
```diff
@@ -1,52 +1,314 @@
...
+						</li>                                
+					</ul>
+			</section>
+
+			<section class="Works">
+				<div class="work_item">
+					<img src="images/Snapshoot Portfolio.png" alt="snapshot portifolio image">
+					<div class="work_details">
+						<div class="project">
+							<p class="work_item_header">Tonic</p>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: This should be a `hx` tag. A `p` tag is not semantically appropriate. Please change it to a heading

File: index.html [hunk 3]
```diff
@@ -1,52 +1,314 @@
...
+							<p class="html">
+								<span>html</span>
+							</p>
+							<p class="css">
+								<span>css</span>
+							</p>
+							<p class="js">
+								<span>javascript</span>
+							</p>
+						</div>
```
Comment (bienvenuushindi) [inline index.html]: - :red_circle: The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be wrapped inside a list ul and each item of them in a li tag. Please replace the `div` container with a `ul` container

File: index.html [hunk 4]
```diff
@@ -1,52 +1,314 @@
...
+				</div>
+			</section>
+			<section class="about_myself">
+				<div class="about_me_container">
+					<div class="about_me">
+						<h1>About<br>Myself</h1>
+						<p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+						<div class="connect">
+							<div class="connect_text2">
+								<p>LET'S CONNECT</p>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: LET'S CONNECT is a title, It should be in a header tag hx. Please make sure you use a header tag hx here

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @c00p75 ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linter checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- **Please make sure your commit messages are limited to 80 characters.**

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by bienvenuushindi — COMMENTED** [review]
**[OPTIONAL] Please in the future project make sure your commit messages are limited to 80 characters.**

**Comment by c00p75** [comment]
Used `<h2`> for project card headers and 'LET'S CONNECT' and wrapped project technologies in list

**Review by OyePriscilla — APPROVED** [review]
# Hi @c00p75  😄 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 



## Highlights
You documented your work professionally. ✔
GitHub flow is used correctly. ✔
All linter checks are ok. ✔


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @oyepriscilla in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**